provider/cluster/kube/client.go

```diff
@@ -28,6 +28,12 @@ import (
 	mtypes "github.com/ovrclk/akash/x/market/types"
 )
 
+var (
+	ErrNoDeploymentForLease = errors.New("kube: no deployments for lease")
+	ErrNoIngressForLease    = errors.New("kube: no ingress for lease")
+	ErrInternalError = errors.New("kube: internal error")
```

Comment:
formatting looks off - is your editor not aligning `=`?  we should have a linter for this.